scheduler/pkg/agent/client.go

```diff
@@ -474,12 +477,13 @@ func (c *Client) StartService() error {
 
 		c.logger.Infof("Received operation")
 
+		timestamp := time.Now().UnixMilli()
```

Comment:
Something to discuss: UnixMilli will return a wall-clock time, which is not monotonic. In particular, it goes back when NTP adjusts the machine's time. So we might see some reordering/out-of-order messages because of that, when the actual events were not out-of-order.

The typical NTP adjustements can get into the 100 - 200 ms range (in principle NTP still works/can make adjustements for large drifts in the tens of second range, but I think those are unlikely to occur frequently)

Reply:
good point. I can just use a counter internally instead of `UnixMilli`

Reply:
In fact `time.Now` contains a monotonic clock, will use that instead.

Reply:
fixed in [9265c30](https://github.com/SeldonIO/seldon-core/pull/5969/commits/9265c30632b83b8eeabb8ea0d2b7f367027d66bb)